### DISCUSSION TITLE
custom directive in ssr mode

### DISCUSSION THREAD

**Discussion by zeynab-rs** [quasarframework/quasar#11274] [General - Components / Directives / etc] [answered]
How to define a custom directive in ssr mode?

**Comment by MilosPaunovic** [comment]
[https://quasar.dev/quasar-cli/developing-ssr/vue-ssr-directives#introduction](https://quasar.dev/quasar-cli/developing-ssr/vue-ssr-directives#introduction)

**Reply by zeynab-rs** [reply]
That's not complete

**Comment by zeynab-rs** [comment, marked as answer]
I wrote my custom directive in boot. That's point was using not ssrContext condition.

boot/my-dir.js:
```
export default ({ app, ssrContext }) => {
  if(!ssrContext){
     //
  }
}
```

And based on quasar document add my-dir.js file in src-ssr/directives folder too:
```
export default (dir) => {
  return {
    props: []
  }
}
```
